chunkflow/chunk/base.py

```diff
@@ -378,17 +378,26 @@ def crop_margin(self, margin_size: tuple = None, output_bbox: Bbox=None):
             assert output_bbox is not None
             return self.cutout(output_bbox.to_slices())
     
-    def threshold(self, threshold: float):
-        seg = self > threshold
+    def threshold(self,
+                  threshold: float,
+                  remap_high: int = None,
+                  remap_low: int = None) -> np.ndarray:
+        # Threshold
+        seg = (self > threshold).astype(np.uint8)
+
+        # Optional remapping
+        if remap_high is not None:
+            assert remap_high >= 0 and remap_high < 256
```

Comment:
You can be extra generic and use `numpy.iinfo` to figure out the the min and max if you want, but since you explicitly want uint8 already it does not matter.